CHANGES.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## 6.3 (2024-04-25)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/6.2...6.3)
```

Comment:
Oops! `CHANGES.md` should probably be excluded from the broken link checker...

Reply:
Done in 50735e5a28660cc6f2d2c472dcd0d6f2bf440c94. Sorry!

Reply:
Not a worry! And thank you for the prompt fix :heart: